modules/ardrivo/include/ardrivo/Error.hxx

```diff
@@ -19,12 +19,18 @@
 #ifndef SMARTCAR_EMUL_ERROR_HXX
 #define SMARTCAR_EMUL_ERROR_HXX
 
+/**
+* Thorws a runtime error
```

Comment:
thorws = throws, but no this functions' sole existence is not just throwing.
On the contrary it creates a runtime error from the given message only if errors are not silenced